core/shared/src/main/scala-3/me/shadaj/scalapy/py/Dynamic.scala

```diff
@@ -0,0 +1,28 @@
+package me.shadaj.scalapy.py
+
+import scala.language.dynamics
+
+import me.shadaj.scalapy.readwrite.FacadeReader
+import me.shadaj.scalapy.interpreter.CPythonInterpreter
+
+@native class Dynamic extends Any with AnyDynamics
+
+object Dynamic {
```

Comment:
This companion object seems to be duplicated with Scala 2, we should use shared code.

Reply:
Is `@native class Dynamic` facade? If yes then can we add it to `core/shared/src/main/scala-2/me/shadaj/scalapy/py/Facades.scala` ?

Reply:
Yep, the `Dynamic` trait/class is just a facade like everything else!

Reply:
I guess we can't split `Dynamic` `class` and `object`, because Scala requires that companion object mus be defined in the same source file

Reply:
> I guess we can't split `Dynamic` `class` and `object`, because Scala requires that companion object mus be defined in the same source file

I think so. I also tried it before and such an error was thrown.

Reply:
Ah, okay, in that case we can just have the companion object for both Scala versions mix in a common trait?

Reply:
> Ah, okay, in that case we can just have the companion object for both Scala versions mix in a common trait?

OK!